dotcom-rendering/src/client/userFeatures/userBenefitsApi.ts

```diff
@@ -0,0 +1,41 @@
+import { isObject } from '@guardian/libs';
+import {
+	getOptionsHeadersWithOkta,
+	type SignedInWithCookies,
+	type SignedInWithOkta,
+} from '../../lib/identity';
+import { fetchJson } from './fetchJson';
+import type { UserBenefits } from './user-features';
+
+type UserBenefitsResponse = {
+	benefits: string[];
+};
+export const syncDataFromUserBenefitsApi = async (
+	signedInAuthStatus: SignedInWithOkta | SignedInWithCookies,
+): Promise<UserBenefits> => {
+	const response = await fetchJson(
+		`${
+			window.guardian.config.page.userBenefitsApiUrl ??
+			'https://user-benefits.code.dev-guardianapis.com/benefits/me'
+		}/me`,
+		{
+			mode: 'cors',
+			...getOptionsHeadersWithOkta(signedInAuthStatus),
+		},
+	);
+	if (!validateResponse(response)) {
+		throw new Error('invalid response');
```

Comment:
presumably that will throw an alarm somewhere for us/sentry?

Reply:
I guess? Not sure, this is the same as the previous implementation

Reply:
true I guess the more pertinent question is how will you know that everything is working with the new API like it was before across a similar percentage of browsers